pkgs/development/interpreters/perl/default.nix

```diff
@@ -1,11 +1,11 @@
 { config, lib, stdenv, fetchurl, pkgs, buildPackages, callPackage
-, enableThreading ? stdenv ? glibc, coreutils, makeWrapper
+, enableThreading ? true, coreutils, makeWrapper
 }:
 
 with lib;
 
 # We can only compile perl with threading on platforms where we have a
-# real glibc in the stdenv.
+# real glibc in the stdenv or on Darwin (due to defaulting to clang).
```

Comment:
This implies it would build without GCC on other platforms if using Clang.

Reply:
Good point. Will try to reword.